lib/auth/webauthncli/api.go

```diff
@@ -57,7 +57,7 @@ type LoginPrompt interface {
 	// PromptTouch prompts the user for a security key touch.
```

Comment:
Could you mention in the type-level LoginPrompt comment that prompts can have remote implementations, thus all methods may error?

I suppose the same will be true for RegisterPrompt at some point, so we should state the same there when the time comes.

Reply:
I also added the comment to `RegisterPrompt` as well anyways (I can remove it if you want)